### DISCUSSION TITLE
File path structure [question]

### DISCUSSION THREAD

**Discussion by fernan2jr** [Thinkmill/keystatic#1158] [Q&A]
Hi, 👋
I'm using Astro for my project and I'm creating a categories collection. Each entry will have a title, slug, and an image selection field for an icon. I know that by default, Keystatic creates a slug folder for each entry and places the image inside it. But what if I don't want the slug folder because I want every icon file to be just in `src/icons`? Is it possible, or am I wrong about what I want?

**Comment by FrancocDev** [comment]
+1 
I'm trying to do the same right now. Did you find a solution?

**Reply by fernan2jr** [reply]
Unfortunately no.
I really wanted to use Keystatic for my project but I also had issues creating custom widgets and nobody responds in any of the support channels.
I ended up going with a different cms.

**Comment by Matthijz98** [comment]
I think you could use the import alias described here: https://keystatic.com/docs/recipes/astro-images
I have only done this for images in the text editor but think it should work for front-mater images as well.
